packages/styles/scss/components/pricing-table/_pricing-table.scss

```
@@ -152,6 +152,16 @@
       display: flex;
       justify-content: space-between;
     }
+
+    .#{$prefix}--pricing-table-cell-content {
+      display: flex;
+      align-items: start;
+      line-height: 1.25rem;
+
+      svg {
+        margin-right: $spacing-03;
```

Consider using the [logical property](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) here. This is especially relevant given our target consumer supports RTL languages.

```suggestion
        margin-inline-end: $spacing-03;
```